### PR TITLE
Added the client side security measures

### PR DESCRIPTION
**Pull request by silkychoudhary**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR closes #< insert number here >.

### CODE REVIEW

File: Document/0x05a-Platform-Overview.md [hunk 1]
```diff
@@ -643,3 +643,5 @@ The Android application attack surface consists of all components of the applica
   - [Data Storage on Android](0x05d-Testing-Data-Storage.md#data-storage-on-android "Data Storage on Android")
 - Protect itself against compromised environments, repackaging or other local attacks, see also:
   - [Android Anti-Reversing Defenses](0x05j-Testing-Resiliency-Against-Reverse-Engineering.md#android-anti-reversing-defenses "Android Anti-Reversing Defenses")
+- Implement the client side security measures such as Root Detection, Certificate/Public Key Pinning
```
Comment (cpholguera) [inline Document/0x05a-Platform-Overview.md:646]: For me this is implied in the previous lines.

File: Document/0x05b-Basic-Security_Testing.md [hunk 2]
```diff
@@ -2144,6 +2144,9 @@ For information on disabling SSL Pinning both statically and dynamically, refer
 - Magisk Modules repository - <https://github.com/Magisk-Modules-Repo>
 - Magisk Trust User Certs module - <https://github.com/NVISO-BE/MagiskTrustUserCerts/releases>
 - Mitm-relay - <https://github.com/jrmdev/mitm_relay>
+- Mitm Proxy - <https://mitmproxy.org>
+- Upstream Proxy - <https://github.com/inaz2/proxy2>
+- JWT Secret Key Cracker - <https://github.com/brendan-rius/c-jwt-cracker>
```
Comment (cpholguera) [inline Document/0x05b-Basic-Security_Testing.md:2149]: This is the references section. Unfortunately we cannot add references to tools without actually referencing them in the chapter. If you think that some tools are missing or worth including please feel free to do so, maybe another chapter is more appropriate:
- https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md
- https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05g-Testing-Network-Communication.md

**Comment by cpholguera** [comment]
Given the comments we'll close the PR but please feel free to open a new one with new content for the mentioned chapters. Also gladly take a look at our issues and pick up one that you like. Thank you!

File: Document/0x05a-Platform-Overview.md (hunk 1 above)
Comment (silkychoudhary) [reply Document/0x05a-Platform-Overview.md:646]: Yes, but it is not explicitly mentioned. It should be there for new security enthusiasts.

Comment (cpholguera) [reply Document/0x05a-Platform-Overview.md:646]: We understand your concern and thanks for pointing that out but we really believe this is already well covered, e.g. talking about "protecting against compromised environments", implies talking about root detection. The corresponding links will lead the reader to the desired information.

**Pull request closed without merging**